spec/unit/recipes/_configure_indexes_spec.rb

```diff
@@ -222,4 +268,88 @@
       expect(subject).to create_splunk_template('system/indexes.conf').with(expected_attributes)
     end
   end
+
+  context 'when remotePath is set in default stanza' do
```

Comment:
minor, "when remotePath is set in default stanza and _maxDailyDataSizeMB is given"

Reply:
updated, Thanks! [273d89b](https://github.com/cerner/cerner_splunk/pull/241/commits/273d89beef2330782b1fa1e4360b0ab359130995#diff-2dbf75fe595324af1923d6c51037b9e78cab1adcc9d842af08ba7e0efba026d4R272)